### DISCUSSION TITLE
Can OpenPGP Authentication subkeys be used?

### DISCUSSION THREAD

**Discussion by Mikaela** [FiloSottile/age#288] [Q&A] [answered]
Can GPG Authentication Subkeys be used as their public versions work for SSH or do they go under ssh-agent & YubiKeys being unusable?

```
└┌(%:~)┌- gpg --export-ssh-key "69FF 455A 869F 9031 A691  E0F1 9939 2F62 BAE3 0723"
ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAILGSHRVWaw4j1WXik/HQ2/m00NjKZLFeSxX4NEBRqevn openpgp:0xDE2F9C12
```

This is very close to https://github.com/FiloSottile/age/discussions/280 and I was originally going to ask there, but I didn't find a comment button or similar and didn't want to post my question as an "answer".

**Comment by FiloSottile** [comment, marked as answer]
I’m interpreting the question as “if I encrypt to an SSH public key with a PGP authentication subkey held on disk as its private key, can I decrypt that file?”

The answer is yes, in theory, but there is no code to do it (yet).

Once plugins are supported, this can be implemented as a plugin.

**Reply by Mikaela** [reply]
Thank you :)
